projects/packages/my-jetpack/_inc/components/product-cards-section/boost-card/index.tsx

```diff
@@ -21,7 +22,14 @@ const BoostCard: FC< { admin: boolean } > = props => {
 
 	const handleMouseLeave = useCallback( () => {
 		setShouldShowTooltip( false );
-	}, [ setShouldShowTooltip ] );
+		if ( forceHideTooltip ) {
+			setForceHideTooltip( false );
+		}
```

Comment:
This way of hiding the tooltip on click is a big wonky 🤔 When testing it I had a few instances where I clicked to dismiss it and just barely went outside the box and back in and had to do it again. It's also not clear that you **can** click on it to dismiss it. Is there a way to get the popover to prefer being above the letter grade rather than to the left of it in this circumstance?

Reply:
Good point, and yes, there is. Should we keep that logic I've added?

Reply:
We *could* keep it and I think it would be okay 🤔, however if we keep the logic for that one and the rest of them don't work that way, it might be a little weird. If you fix the issue by moving the tooltip above the letter grade I think I'd lean toward removing this logic but I'll leave it up to your discretion

Reply:
I've gone ahead and just removed that to keep things clean!